<launch>
    <node name="stack_control" pkg="isaac_sim_zone_SA" type="robot_tasks.py" output="screen"/>
    <node name="monitor_node" pkg="isaac_sim_zone_SA" type="Monitor.py" output="screen"/>
    <node name="control_node" pkg="isaac_sim_zone_SA" type="Control.py" output="screen"/>
    <group ns="carter1">
        <param name="map_yaml_path" value ="home/russell/thesis_ws/src/navigation_SA/isaac_sim_zone_SA/assets/carter_LEfloor_navigation.yaml" />
        <param name="iteration_count" type="int" value="50000" />
        <param name="goal_generator_type" type="str" value="GoalReader" />
        <param name="action_server_name" type="str" value="move_base" />
        <param name="obstacle_search_distance_in_meters" type="double" value="0.2" />
        <param name="goal_text_file_path" value="/home/russell/thesis_ws/src/navigation_SA/isaac_sim_zone_SA/assets/goals1.txt" />
        <rosparam param="initial_pose">[22.0, 3.0, 0, 0, 0, 1, 0] </rosparam>
        <node name="set_navigation_goal" pkg="isaac_sim_zone_SA" type="set_goal.py" output="screen"/>
    </group>
    <group ns="carter2">
        <param name="map_yaml_path" value="home/russell/thesis_ws/src/navigation_SA/isaac_sim_zone_SA/assets/carter_LEfloor_navigation.yaml" />
        <param name="iteration_count" type="int" value="50000" />
        <param name="goal_generator_type" type="str" value="GoalReader" />
        <param name="action_server_name" type="str" value="move_base" />
        <param name="obstacle_search_distance_in_meters" type="double" value="0.2" />
        <param name="goal_text_file_path" value="/home/russell/thesis_ws/src/navigation_SA/isaac_sim_zone_SA/assets/goals2.txt" />
	<rosparam param="initial_pose">[9.0, 84.0, 0, 0, 0, 1, 0] </rosparam>
        <node name="set_navigation_goal" pkg="isaac_sim_zone_SA" type="set_goal.py" output="screen"/>
    </group>
    <group ns="carter3">
        <param name="map_yaml_path" value="home/russell/thesis_ws/src/navigation_SA/isaac_sim_zone_SA/assets/carter_LEfloor_navigation.yaml" />
        <param name="iteration_count" type="int" value="50000" />
        <param name="goal_generator_type" type="str" value="GoalReader" />
        <param name="action_server_name" type="str" value="move_base" />
        <param name="obstacle_search_distance_in_meters" type="double" value="0.2" />
        <param name="goal_text_file_path" value="/home/russell/thesis_ws/src/navigation_SA/isaac_sim_zone_SA/assets/goals3.txt" />
        <rosparam param="initial_pose">[70.0, 3.0, 0, 0, 0, 1, 0] </rosparam>
        <node name="set_navigation_goal" pkg="isaac_sim_zone_SA" type="set_goal.py" output="screen"/>
    </group>
</launch>


